Week2/homework/Array.js

```diff
@@ -0,0 +1,57 @@
+//Write a JavaScript function to get the first element of an array.
+
+const arr = [1, 2, 3, 4];
+const getFirstElement = arr => arr.shift();
```

Comment:
getFirstElement & getFirstNElements functions should be a single function. Try to use default parameter for the function